lib/portage/sync/syncbase.py

```diff
@@ -294,6 +300,24 @@ def noisy_refresh_keys():
 				loop.run_until_complete(decorated_func())
 		out.eend(0)
 
+	def _get_openpgp_env(self, openpgp_key_path=None):
+		if gemato:
```

Comment:
I would prefer an explicit:

```suggestion
		if gemato is not None:
```